tests/README.md

```diff
@@ -0,0 +1,26 @@
+## How to get started
+
+Python 3.6 is required.
+For settting up Python 3 environment, [virtualenv](https://virtualenv.pypa.io/en/stable/) is commonly used, like:
+
+```
+# once
+virtualenv -p python3 venv
+
+# each session
+. venv/bin/activate
+```
+
+Then install required packages:
+
+```
+pip install -r requirements.txt
+```
+
+## Running the tests
```

Comment:
Instruction to select one test or just one functions is useful.
```
pytest test_erc721.py
```
```
pytest test_erc721.py::test_approve
```

Reply:
yup!